telegraf/plan.sh

```diff
@@ -0,0 +1,59 @@
+pkg_name=telegraf
+pkg_origin=core
+pkg_version="1.3.1"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('The MIT License (MIT)')
+pkg_description="telegraf - client for InfluxDB"
+pkg_upstream_url="https://github.com/influxdata/telegraf/"
+pkg_scaffolding="core/scaffolding-go"
+pkg_svc_run="bin/telegraf"
+pkg_bin_dirs=(bin)
+
+do_begin() {
+    export GOBIN="${GOPATH}/bin"
+}
+
+do_download() {
+    git clone --branch "${pkg_version}" https://github.com/influxdata/telegraf || true
+    cd telegraf && \
+        git pull origin $pkg_version
+}
+
+do_verify() {
+    do_default_verify
+}
+
+do_clean() {
```

Comment:
since its coming from git, it needs to skip this step, otherwise:
```
+ do_clean
+ do_default_clean
+ scaffolding_go_clean
+ local clean_args
+ clean_args='-r -i'
+ [[ -n '' ]]
+ build_line 'Clean the cache'
+ [[ '' == \t\r\u\e ]]
+ case "${TERM:-}" in
+ echo -e '   \033[1;36mtelegraf: \033[1;37mClean the cache\033[0m'
   telegraf: Clean the cache
+ return 0
+ pushd /hab/cache/src/telegraf-1.3.1/src/
/hab/pkgs/core/scaffolding-go/0.1.0/20170711155330/lib/scaffolding.sh: line 71: pushd: /hab/cache/src/telegraf-1.3.1/src/: No such file or directory
++ _on_exit
++ local exit_status=1
```